conf/registry-oracle-example.yaml

```diff
@@ -67,6 +67,11 @@ swagger:
 #enable CORS, may want to disable in production
 enableCors: false
 
+#Set below 3 properties if server needs a proxy to connect. Useful to download mysql jar
```

Comment:
nit : i dont think we should reference mysql in registry-oracle-example.yaml

Reply:
Addressed